docs/requirements.md

```diff
@@ -1,51 +1,143 @@
 
-With this in mind, you must perform all operations or transactions against a single master instance so that you can achieve linearizability and avoid worrying about consistency issues in your application. In other words, ScalarDB does not support read replicas. 
+#### Analytics
+{:.no_toc}
+- Not applicable. Cassandra always returns committed records, so there are no Cassandra-specific requirements.
+
+</div>
+</div>
+
+## Recommendations
+
+Properly configuring each underlying database of ScalarDB for high performance and high availability is recommended. The following recommendations include some knobs and configurations to update.
 
 {% capture notice--info %}
 **Note**
 
-You can still use a read replica as a backup and standby even when following this guideline.
+ScalarDB can be seen as an application of underlying databases, so you may want to try updating other knobs and configurations that are commonly used to improve efficiency.
 {% endcapture %}
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+<div id="tabset-2">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'JDBC_databases2', 'tabset-2')" id="defaultOpen-2">JDBC databases</button>
+  <button class="tablinks" onclick="openTab(event, 'DynamoDB2', 'tabset-2')">DynamoDB</button>
+  <button class="tablinks" onclick="openTab(event, 'Cosmos_DB_for_NoSQL2', 'tabset-2')">Cosmos DB for NoSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'Cassandra2', 'tabset-2')">Cassandra</button>
+</div>
+
+<div id="JDBC_databases2" class="tabcontent" markdown="1">
+- Use read-committed isolation for better performance.
+- Follow the performance optimization best practices for each database. For example, increasing the buffer size (for example, `shared_buffers` in PostgreSQL) and increasing the number of connections (for example, `max_connections` in PostgreSQL) are usually recommended for better performance.
+</div>
 
+<div id="DynamoDB2" class="tabcontent" markdown="1">
+- Increase the number of read capacity units (RCUs) and write capacity units (WCUs) for high throughput.
+- Enable point-in-time recovery (PITR).
+
+{% capture notice--info %}
+**Note**
+
+Since DynamoDB stores data in multiple availability zones by default, you don’t need to adjust any configurations to improve availability.
+{% endcapture %}
 <div class="notice--info">{{ notice--info | markdownify }}</div>
+</div>
+
+<div id="Cosmos_DB_for_NoSQL2" class="tabcontent" markdown="1">
+- Increase the number of Request Units (RUs) for high throughput.
+- Enable point-in-time restore (PITR).
+- Enable availability zones.
+</div>
+
+<div id="Cassandra2" class="tabcontent" markdown="1">
+- Increase `concurrent_reads` and `concurrent_writes` for high throughput. For details, see the official Cassandra documentation about [`concurrent_writes`](https://cassandra.apache.org/doc/stable/cassandra/configuration/cass_yaml_file.html#concurrent_writes).
+</div>
+</div>
```

Comment:
Similar to [my previous comment](), can we change the order of the tabs as follows?:

- Cassandra
- Cosmos DB for NoSQL
- DynamoDB
- JDBC databases

```suggestion
<div id="tabset-2">
<div class="tab">
  <button class="tablinks" onclick="openTab(event, 'Cassandra2', 'tabset-2')" id="defaultOpen-2">Cassandra</button>
  <button class="tablinks" onclick="openTab(event, 'Cosmos_DB_for_NoSQL2', 'tabset-2')">Cosmos DB for NoSQL</button>
  <button class="tablinks" onclick="openTab(event, 'DynamoDB2', 'tabset-2')">DynamoDB</button>
  <button class="tablinks" onclick="openTab(event, 'JDBC_databases2', 'tabset-2')">JDBC databases</button>
</div>

<div id="Cassandra2" class="tabcontent" markdown="1">
- Increase `concurrent_reads` and `concurrent_writes` for high throughput. For details, see the official Cassandra documentation about [`concurrent_writes`](https://cassandra.apache.org/doc/stable/cassandra/configuration/cass_yaml_file.html#concurrent_writes).
</div>

<div id="Cosmos_DB_for_NoSQL2" class="tabcontent" markdown="1">
- Increase the number of Request Units (RUs) for high throughput.
- Enable point-in-time restore (PITR).
- Enable availability zones.
</div>

<div id="DynamoDB2" class="tabcontent" markdown="1">
- Increase the number of read capacity units (RCUs) and write capacity units (WCUs) for high throughput.
- Enable point-in-time recovery (PITR).

{% capture notice--info %}
**Note**

Since DynamoDB stores data in multiple availability zones by default, you don’t need to adjust any configurations to improve availability.
{% endcapture %}
<div class="notice--info">{{ notice--info | markdownify }}</div>
</div>

<div id="JDBC_databases2" class="tabcontent" markdown="1">
- Use read-committed isolation for better performance.
- Follow the performance optimization best practices for each database. For example, increasing the buffer size (for example, `shared_buffers` in PostgreSQL) and increasing the number of connections (for example, `max_connections` in PostgreSQL) are usually recommended for better performance.
</div>
</div>
```

Reply:
@josh-wong  I think setting `Cassandra` as the default is a little weird because it looks like it is most used and recommended, but actually, it's not really used.
So, the other docs should also be updated.
I'm okay with making this tab have the same order as the other docs for now and reordering all the tabs at once, but since this doc is already entirely reorganized, I changed the order in this PR.

@brfrn169 What do you think?

Reply:
Right now, there's no reason for Cassandra to be set as the default. I feel the order in this PR (JDBC -> Dynamo -> Cosmos -> Cassandra) makes more sense given our current situation.

Reply:
I see. I'm guessing the order of the databases in other docs was based on alphabetical order, so I didn't think they would need to be reorganized based on use case or popularity.

I'm OK with leaving this as is and then updating the existing tabs in other docs to follow the same order (which interestingly enough is reverse-alphabetical order, haha).